waspc/ChangeLog.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [WIP] 0.12.0
```

Comment:
This is a new thing I started doing: adding "WIP" in front of the release in Changelog that is not yet released.

Reply:
0.12.0 means that we should be careful with the starter templates -> keep the dynamic version until we solve the versioning

Reply:
Aha, because they would be on 0.12 and would not work for previous versions? OK sure! I am sure we will be taking this into consideration when we will handle the versioning of starter templates.